src/supabase/queries/cases.ts

```diff
@@ -94,6 +122,48 @@ export function parseCase(item: any): Case {
   };
   return formattedCase;
 }
+/**
+ * Update a specific User/Case status
+ * @param caseId specified caseId
+ * @param status status to be updated in the specific User/Case row
+ * @returns nothing
+ */
+export async function updateCaseStatus(
+  caseId: CaseUid,
+  status: Eligibility,
+): Promise<void> {
+  try {
+    const {
+      data: { user },
+    } = await supabase.auth.getUser();
+    const userId = user?.id;
+    const { error } = await supabase
+      .from('status')
+      .update({ eligibility: status })
+      .eq('userId', userId)
+      .eq('caseId', caseId);
+    console.log(error);
```

Comment:
error is caught in the try catch so we can remove this console.log here and log it in the `catch` statement